lib/pcr.c

```diff
@@ -218,6 +220,44 @@ bool pcr_print_pcr_struct(TPML_PCR_SELECTION *pcrSelect, tpm2_pcrs *pcrs) {
 }
 
 
+bool pcr_print_pcr_selections(TPML_PCR_SELECTION *pcr_selections) {
+    tpm2_tool_output ("selected-pcrs:\n");
+
+    /* Iterate throught the pcr banks */
+    for (UINT32 i = 0; i < pcr_selections->count; i++) {
```

Comment:
older versions of GCC have issues with this, we learned this the hard way on Android. Go back to not declaring them in the loop bodies, just declare it right above it.